### ISSUE TITLE
Asymmetric Loss For Multi-Label Classification

### ISSUE THREAD

**Issue by shpotes** [shpotes/tensorflowers#20]

**Comment by shpotes** [comment]
When training, [overfit all recorded losses](https://wandb.ai/tensorflowers/challenge/runs/2c286xcy?workspace=user-shpotes). However, despite this, it gives the best results on the validation set. Surely it will be necessary to add some regularizations.

**Comment by shpotes** [comment]
It's a good contender, worth considering in the final tunning
